script.js

```diff
@@ -24,8 +24,14 @@ isChrome = () => {
 };
 
 headersReceivedListener = (requestDetails) => {
+  // Do not count bytes from requests from local cache
+  if (requestDetails.fromCache) return
+
+  let CONST_INCOGNITO = 'Incognito';
```

Comment:
Should be a const I think?
```suggestion
  const CONST_INCOGNITO = 'Incognito';
```